esp-hal/src/mcpwm/mod.rs

```diff
@@ -22,7 +22,12 @@
 //!       registers with flexible updating methods.
 //! * Fault Detection Module (Not yet implemented)
 //! * Capture Module (Not yet implemented)
-//!
+#![doc = ""]
```

Comment:
alternative would be to document it for all chips - I guess having just the right information for the currently selected chip is better?

Reply:
Yeah I've been wondering what we should do about this as well. I guess in this case documenting it on a per-chip basis is simple enough, so this should be fine.

Reply:
This is the nicest way to do it I've seen so far, so I'm going to steal this method :D. Thanks!

Reply:
FYI it doesn't add any newlines, so you can just inline the bit you want to change, for example:

```rust
//! The System Timer is a 
#![cfg_attr(esp32s2, doc = "`64-bit`")]
#![cfg_attr(not(esp32s2), doc = "`54-bit`")]
//! timer with three comparators capable of signalling an alarm interupt on each.
```
Creates

![image](https://github.com/esp-rs/esp-hal/assets/6977289/ce6d0fde-979a-4496-9a49-e06d2f5a5ccc)

Which is nice.